scripts/qwen-review.sh

```
@@ -0,0 +1,303 @@
+  esac
+done
+
+require_cmd curl    "macOS ships curl — check PATH"  || exit 2
+require_cmd python3 "python3 is required for JSON I/O" || exit 2
+require_cmd git     "install git" || exit 2
+
+if ! is_inside_git_repo; then
+  fail "Not a git repository — run from the target repo root."
+  exit 2
+fi
+cd "$(repo_root)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)